index.rst

```diff
@@ -372,9 +375,10 @@ These tests can also be dangrous in parallel execution, as they can be subject t
 Clobbering and Skipping Outputs
 -------------------------------
 
-Some command-line tasks (especially the ``*Driver.py`` tasks) test whether a data product exists in the current rerun chain, and skip any processing that would be replace it.  This is exactly the behavior desired when a large job dies unexpected and you want to resume it.  But it can be very confusing when you actually want to re-do the processing (especially the fact that processing is skipped if the output data product appears anywhere in the rerun *chain*, not just the last rerun in the chain -- this is another behavior we plan to change in the future).
-
-Tasks with this behavior have configuration parameters to disable it, usually with names with words like "overwrite", "clobber", or "skip".  Because these are configuration parameters (not normal command-line options), changing them and then restarting processing in the same rerun will trigger an error of the type described in the :ref:`previous section <configuration-and-software-version-changes>`.
+The ``multibandDriver.py`` and ``coaddDriver.py`` scripts can test whether a data product exists in the output repository, and skip any processing that would be replace it.
+This is exactly the behavior desired when a large job dies unexpected and you want to resume it, but it can be very confusing when you actually want to re-do the processing, so it's not enabled by default.
+To turn it on, pass ``--reuse-outputs-from all`` on the command line.
+You can also pass the name of a subtask (e.g. ``mergeCoaddDetections`` for ``multibandDriver.py``) instead of ``all`` to consider skipping only up to (and including) that step.
```

Comment:
I find this confusing. Perhaps "to allow skipping" instead of "to consider skipping". Or even consider "to skip existing outputs". Also I suggest removing the parenthesis from "(and including)"; use nothing or commas instead, or consider "only up through that step" (I think "through" is sufficiently clear, but your wording is unmistakable).

Reply:
I've switched to "to allow skipping that step *and* any previous steps".